app/stacks/uk-west/appeals-service/cosmosdb.tf

```diff
@@ -73,3 +73,34 @@ resource "azurerm_private_endpoint" "cosmosdb" {
 
   tags = local.tags
 }
+
+import {
+  to = azurerm_cosmosdb_mongo_database.appeals_cosmosdb
+  id = "/subscriptions/962e477c-0f3b-4372-97fc-a198a58e259e/resourceGroups/pins-rg-${local.service_name}-${local.resource_suffix}/providers/Microsoft.DocumentDB/databaseAccounts/pins-cosmos-${local.service_name}-${local.resource_suffix}/mongodbDatabases/${local.forms_web_app_mongodb}"

```

Comment:
This will only work in dev, as the subscription ID changes. You can use `data.azurerm_client_config.subscription_id` 

(https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/client_config#subscription_id)

Reply:
Ah awesome thanks, I couldn't work out where that uuid came from 😅